### DISCUSSION TITLE
SchemaForm value不更新问题

### DISCUSSION THREAD

**Discussion by Gamehu** [alibaba/formily#1487] [Q&A 1.x] [answered]
操作步骤：
1.多次手动清空“默认值”输入框，
2.点击【保存】按钮，重新设置value，不生效
 
[示例](https://codesandbox.io/s/updatedata-sq6fn?file=/index.js)

**Comment by janryWang** [comment, marked as answer]
你这是受控更新value，因为props没变化，重新更新是不生效的，可以加个时间戳来强制触发变化
